client/src/component/itemList.test.js

```diff
@@ -0,0 +1,19 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import ItemList from './itemList';
+
+describe('itemList test', () => {
+    const fn = jest.fn();
+    render(<ItemList ticketItems={fn} />);
```

Comment:
Isn't the prop called setTicketItems? That's what you can test is called in the button click below.

Reply:
The button click renders a new item component. The setTicketItem prop is called in the useEffect hook when there is a change in the itemList. If I'm understanding this correctly, the button click should be testing if the function for rendering a new item is being called, right?

Reply:
It just looks like the prop used here isn't used in the actual component.

Reply:
Ah okay.  Should I change the prop then or is something like this okay?